dashboard_legacy/middleware/files_api.rb

```diff
@@ -177,10 +177,9 @@ def record_event(quota_event_type, quota_type, encrypted_channel_id)
   # Only from codeprojects.org domain
   # Deprecated in favor of the URL below
   #
-  get %r{/([^/]+)/([^/]+)$}, {code_projects_domain: true} do |encrypted_channel_id, filename|
+  get %r{/([^/]+)/$}, {code_projects_domain: true} do |encrypted_channel_id|
     pass unless valid_encrypted_channel_id(encrypted_channel_id)
-
-    get_file('files', encrypted_channel_id, filename, true)
+    redirect "/projects/weblab#{request.path_info}"
```

Comment:
now this route looks like a duplicate of the route on line 205. if this change is intentional, perhaps this route can just be removed? if you want to keep it, I think it would be helpful to include an example of the route you are trying to match in the comments.

Reply:
Good call! I deduplicated them